packages/babel-parser/typings/babel-parser.d.ts

```diff
@@ -158,7 +158,8 @@ export interface DecoratorsPluginOptions {
 }
 
 export interface PipelineOperatorPluginOptions {
-  proposal: "fsharp" | "minimal" | "smart";
+  proposal: "minimal" | "fsharp" | "hack" | "smart";
+  topicToken: "#";
```

Comment:
Is `topicToken` required in operator plugin options? Maybe `topicToken?`?

Reply:
`topicToken` is purposefully required (see the documentation in babel/website#2541 and also: https://github.com/babel/babel/blob/11a7a491b57c7c1e2b33cf99a9d244c6eb6e32d6/packages/babel-plugin-syntax-pipeline-operator/src/index.js#L11-L16

`topicToken` will be an option between at least two choices, `#` and `%`, with no particular choice currently considered better than the other. In the future, if TC39 advances Hack pipes, then it will bikeshed on the particular topic token and standardize one of them, and then that token would become the default.

Reply:
…Oh, wait, I’m being silly. Of course `topicToken` is optional…when `proposal` isn’t `"hack"`.

Reply:
Fixed in 01ca985505265ec81ba7be05cb3aa22d208f7021.